test/verify/check-networking-firewall

```diff
@@ -177,6 +178,66 @@ class TestFirewall(MachineCase):
             b.click("caption button.btn-primary")
             b.wait_present("#cockpit_modal_dialog li input[data-id='empty']")
 
+    @skipImage("Custom ports were introduced in 191", "rhel-7-6-distropkg")
+    def testAddCustomServices(self):
+        b = self.browser
+
+        self.login_and_go("/network/firewall")
+
+        # click on the "Add Services" button
+        b.wait_present("caption button.btn-primary:enabled")
```

Comment:
Hm, this doesn't actually click. it does in `open_dialog`, but please just move this into that function.